### PR TITLE
Mobile portfolio project pull request

### PR DESCRIPTION
**Pull request by Ferizad**
Addition of HTML, CSS, and updated README file for my Mobile Portfolio project.

### CODE REVIEW

**Review by Debas-31 — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle:  :red_circle:  :red_circle: 

- [ ] You have submitted a project with Pull Request and should not have a summary describing introduced changes in more detail.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -23,15 +23,21 @@ Feel free to check the [issues page](../../issues/).
...
 ## ⭐️ Show your support <a name="support"></a>
 Feel free to leave comments and contact via given authors information above.
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<<<<<<< HEAD
+=======
 <!-- ACKNOWLEDGEMENTS -->
+>>>>>>> af642373f43c6a0e09147fdb7af75fd4084f44fb
```
Comment (frank1738) [inline README.md:40]: - [ OPTIONAL] Kindly remove unnecessary comments from your readme file to keep it clean and professional

File: README.md [hunk 2]
```diff
@@ -23,15 +23,21 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<<<<<<< HEAD
+=======
 <!-- SUPPORT -->
+>>>>>>> af642373f43c6a0e09147fdb7af75fd4084f44fb
```
Comment (frank1738) [inline README.md:29]: - [ OPTIONAL] Kindly remove unnecessary comments from your readme file to keep it clean and professional

**Review by frank1738 — APPROVED** [review]
Hi @Ferizad , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Design👍

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ferizad** [comment]
"Works" and "mobile section" were added in HTML and CSS files.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+    <section class="about_self_section">
+
+        <div>
+
+            <h3><a class="textdecoration font7" href="#">About Myself</a></h3>
+
+            <p>Hello I'm a software developer! I can help you build a product, feature or website Look through some of
+                my work and experience! if you like what you see and have a project you need coded, don't hesistate to
+                contact me.</p>
+
```
Comment (mugishasam123) [inline index.html:224]: - [ ] Please try to stick to the Figma design as much as possible to make your project looks good. you can do so by leveraging colors, fonts, and font sizes.

Your design:
image

Figma design:
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+                    <ul class="about_section_align1">
+
+                        <li><a href="#"><img src="media/css_icon.jpg" alt=""></a></li>
+                        <li><a class="font5 textdecoration" href="#">CSS</a></li>
+
+                    </ul>
+
+                </li>
+
+                <li>
```
Comment (mugishasam123) [inline index.html:270]: - [ ] Please you should stick to the design as much as possible to make your work more professional, try to match your design with the given one.

Your design:
image

Figma design:
image

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @Ferizad  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors 👍 
- Descriptive PR title 🥇 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ferizad** [comment]
Modified HMTL and CSS files to stick to Figma design. Added vector icons in the last frameworks, and skills part.

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @Ferizad , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good UI design :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ferizad** [comment]
Updated HTML and CSS files to change the font size on the last part according to the Figma template.

**Review by algerina — APPROVED** [review]
Hi @Ferizad , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 


 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**